src/components/Frame/components/Toast/Toast.scss

```diff
@@ -4,7 +4,7 @@ $Backdrop-opacity: 0.88;
 .Toast {
   @include text-style-display-small;
   display: inline-flex;
-  max-width: rem(400px);
+  max-width: rem(720px);
```

Comment:
I'm not sure the width needs to be increased to 720px. If the content guidelines suggest a message no longer than 3 words, does it really need to exceed 500px?

![screen shot 2019-02-12 at 5 32 21 pm](https://user-images.githubusercontent.com/6239480/52680117-35b93300-2eec-11e9-8762-004eb70d5034.jpg)